wiki/Article_Styling_Criteria/en.md

```diff
@@ -28,31 +42,37 @@ Listed below are the properly-supported locales for the osu! wiki.
 | `pt-br.md` | Brazilian Portuguese | Português (Brasil) |
 | `ru.md` | Russian | Русский |
 | `sv.md` | Swedish | Svenska |
+| `th.md` | Thai | ไทย |
+| `tr.md` | Turkish | Türkçe |
+| `vi.md` | Vietnamese | Tiếng Việt |
 | `zh.md` | Chinese (Simplified) | 简体中文 |
 | `zh-tw.md` | Traditional Chinese (Taiwan) | 繁體中文（台灣） |
 
-Keep in mind, it is intended that the osu! wiki will try to give readers their selected language's version of an article (and fallback on using the English version if it is not available). If your language is not listed, you have two choices:
+*Notice: It is intended that the osu! wiki will try to give readers their selected language's version of an article (and fallback on using the English version if it is not available).*
+
+If your language is not listed above, you can either append `?locale={langcode}` to the URL (where `{langcode}` is your language's two letter code in lowercase letters, e.g. `https://osu.ppy.sh/help/wiki/Welcome?locale=zh` will give you the Chinese version of the Welcome article regardless of the selected language) or help translate the [osu-web via Crowdin](https://crowdin.com/project/osu-web). If you need help translating the website or want to discuss translating decisions, you can talk about it in the `#osu-web` [osu!dev Discord channel](https://discord.gg/ppy).
```

Comment:
`` `#osu-web` channel via the [osu!dev Discord](https://discord.gg/ppy) server.``

Change it to whatever, but don't confuse channel with server. Also for other occurrences, make it consistent.

Reply:
https://github.com/ppy/osu-wiki/pull/1334#discussion_r192937570

Eh, didn't know where to put the word "channel"